sql/hive-thriftserver/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
This file is added because when I run the mvn tests for hive-thriftserver module, it is reported missing